static/js/components/LearnerSearch.js

```diff
@@ -128,6 +129,16 @@ export default class LearnerSearch extends SearchkitComponent {
       {openLearnerEmailComposer: this.props.openLearnerEmailComposer},
       LearnerResult
     );
+    this.forceSearch = true;
+  }
+
+  loaded = (): boolean => {
+    if (this.forceSearch) {
```

Comment:
seems like we could also just inline a call like

```js
<Loader loaded={this.forceSearch || !this.isInitialLoading}>
```

in the JSX, and the method could just do

```js
trackRenderedOnce = () => {
  this.forceSearch = false;
}
```

I think you could then just have `trackRenderedOnce` be called in `componentDidMount`, but I'm not sure we have a guarantee that `componentDidMount` is called after exactly one call to `render`, @noisecapella do you know if that would work?